FreeRTOS-Plus/Demo/coreMQTT_Windows_Simulator/MQTT_Multitask/DemoTasks/MultitaskMQTTExample.c

```diff
@@ -2188,8 +2075,9 @@ static void prvMQTTDemoTask( void * pvParameters )
         configASSERT( xMQTTStatus == MQTTSuccess );
         configASSERT( globalMqttContext.connectStatus == MQTTConnected );
 
-        /* Give subscriber task higher priority so the subscribe will be processed before the first publish.
-         * This must be less than or equal to the priority of the main task. */
+        /* Give subscriber task higher priority so the subscribe will be processed
+        before the first publish.  This must be less than or equal to the priority of
+        the main task. */
```

Comment:
I don't think this matches the rest of the style
```suggestion
        /* Give subscriber task higher priority so the subscribe will be processed
         * before the first publish.  This must be less than or equal to the priority of
         * the main task. */
```